aws-integrations/firehose/CHANGELOG.md

```diff
@@ -18,4 +18,7 @@
 * [Update] Made naming & update changes to align Firehose template with Integration team's firehose-metricss one.
 
 ### 0.0.1 / 30 Oct 2023
-* [Update] Fix for bucketname to be short and globally unique by stack ID
\ No newline at end of file
+* [Update] Fix for bucketname to be short and globally unique by stack ID
+
+### 0.0.1 / 02 Nov 2023
```

Comment:
Version should be update and also The CF template SemanticVersion metadata should also be updated